docs/data/stashcache/run-stashcache-container.md

```diff
@@ -0,0 +1,184 @@
+             --volume <HOST PARTITION>:/tmp
+```
+
+The `HOST PORT` is the port on your computer which will accept caching requests and <HOST PARTITION> will be the mount where to store
+the cached files.
+
+
+You can verify that it worked with the command:
+
+```console
+user@host $ curl http://localhost:8212/user/dweitzel/public/blast/queries/query1
+```
+
+Which should output:
+
+```
+>Derek's first query!
+MPVSDSGFDNSSKTMKDDTIPTEDYEEITKESEMGDATKITSKIDANVIEKKDTDSENNITIAQDDEKVSWLQRVVEFFE
+```
+
+Readying for Production
+------------------------
+
+Additional configuration is needed to make StashCache production-ready.
+
+1. Add environment variables to set the name of the cache and the directory of the persistant cache.  It is important to remember that the directory set in the environment variable points to the directory **inside** the container.  It is later mapped to a host directory with the `--volume` option.
+
+An example final `docker run` command:
+```console
+user@host $ docker run --rm --publish <HOST PORT>:8000 \
+             --volume <HOST PARTITION>:/cache \
+             --env-file=/opt/xcache/.env \
+             opensciencegrid/stash-cache:fresh
+```
+
+Where the environment file on the docker host, `/opt/xcache/.env`, has the following contents:
+```file
+XC_RESOURCENAME=ProductionCache
+XC_ROOTDIR=/cache
+```
+
+It is recommended to use a container orchestration service such as [docker-compose](https://docs.docker.com/compose/) or [kubernetes](https://kubernetes.io/), or start the XCache container with systemd.
+
+An example systemd service file for xcache.  This will require creating the environment file in the directory `/opt/xcache/.env`. 
+
+!!! note
+    This example systemd file assumes <HOST PORT> is `8000` and  <HOST PARTITION> is `/srv/cache`.
+
+```file
+[Unit]
+Description=XCache Container
+After=docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+ExecStartPre=-/usr/bin/docker stop %n
+ExecStartPre=-/usr/bin/docker rm %n
+ExecStartPre=/usr/bin/docker pull opensciencegrid/stash-cache:fresh
+ExecStart=/usr/bin/docker run --rm --name %n --publish 8000:8000 --volume /srv/cache:/cache --env-file /opt/xcache/.env opensciencegrid/stash-cache:fresh
+
+[Install]
+WantedBy=multi-user.target
+```
+
+This systemd file can be saved to `/etc/systemd/system/docker.stash-cache.service` and started with:
+
+```console
+root@host $ systemctl enable docker.stash-cache
+root@host $ systemctl start docker.stash-cache
+```
+
+!!! warning
+    [Register](../stashcache/install-cache/#registering-the-cache) the cache before considering it a production service.
+
+Additional Configuration (Optional)
+-----------------------------------
+
+In addition to the configuration avove before starting the container, write the following configuration on your docker host:
+
+1. Write in same file as above(`/opt/xcache/.env`) containing the following required environment  variables and values for your XCache:
+
```

Comment:
Everything above this line in this section seems like it can be replaced with a single sentence:

```
In addition to the required configuration above, you may also configure the behavior of your cache with the following variables in your environment file:
```